src/main/resources/settings.yml.template

```diff
@@ -86,6 +86,7 @@ system:
   customHTMLFiles: false # enable to have files placed in /customFiles/templates override the existing template HTML files
   tessdataDir: /usr/share/tessdata # path to the directory containing the Tessdata files. This setting is relevant for Windows systems. For Windows users, this path should be adjusted to point to the appropriate directory where the Tessdata files are stored.
   enableAnalytics: 'true' # set to 'true' to enable analytics, set to 'false' to disable analytics; for enterprise users, this is set to true
+  disableSanitize: false # set to 'true' to disable Sanitize HTML, set to false to enable Sanitize HTML; (can lead to injections in HTML)
```

Comment:
The order of your comment is confusing since it says
set to false to enable Sanitize HTML; (can lead to injections in HTML)
Which is only true if set to true

I would remove the ``set to false to enable Sanitize HTML;`` since its implied